packages/babel-runtime/package.json

```diff
@@ -701,24 +719,6 @@
       "./helpers/skipFirstGeneratorNext.js"
     ],
     "./helpers/esm/skipFirstGeneratorNext": "./helpers/esm/skipFirstGeneratorNext.js",
-    "./helpers/toPrimitive": [
-      {
-        "node": "./helpers/toPrimitive.js",
-        "import": "./helpers/esm/toPrimitive.js",
-        "default": "./helpers/toPrimitive.js"
-      },
-      "./helpers/toPrimitive.js"
-    ],
-    "./helpers/esm/toPrimitive": "./helpers/esm/toPrimitive.js",
-    "./helpers/toPropertyKey": [
-      {
-        "node": "./helpers/toPropertyKey.js",
-        "import": "./helpers/esm/toPropertyKey.js",
-        "default": "./helpers/toPropertyKey.js"
-      },
-      "./helpers/toPropertyKey.js"
-    ],
-    "./helpers/esm/toPropertyKey": "./helpers/esm/toPropertyKey.js",
```

Comment:
There is no new exports introduced in `babel-runtime`, the export metadata of these two helpers are now placed after `regenerator-runtime`.